packages/jaeger-ui/src/utils/tracking/ga.tsx

```diff
@@ -28,11 +28,11 @@ const isTruish = (value?: string | string[]) => {
 };
 
 const GA: IWebAnalyticsFunc = (config: Config, versionShort: string, versionLong: string) => {
-  const isProd = process.env.NODE_ENV === 'production';
-  const isDev = process.env.NODE_ENV === 'development';
-  const isTest = process.env.NODE_ENV === 'test';
+  const isProd = __APP_ENVIRONMENT__ === 'production';
+  const isDev = __APP_ENVIRONMENT__ === 'development';
+  const isTest = __APP_ENVIRONMENT__ === 'test';
```

Comment:
nit: I would prefer to future-proof this a bit and isolate all env var access to a util module, so that we could change it in one place instead of many files. The "isolate" part can be a standalone PR merged first.